jenkinsci/plot-plugin
# [JENKINS-74098] Remove inline JS script and legacy onClick handler

Jira: [JENKINS-74098](https://issues.jenkins-ci.org/browse/JENKINS-74098)

### What has been done
1. Remove the inline JS <script> tag as well as its corresponding legacy onChange handler to improve CSP compatibility. 


### Screenshots

[Before](https://www.loom.com/share/e3fcf585f17e456ca6e3c6a62d08b66e)
[After](https://www.loom.com/share/d4a17b7cbda647b290c76d3312349f1d)

### How to test
1. Create a freestyle project
2. In `Post build actions` select `Plot build data`
3. In the `Plot Group` enter a name (ex. "test") and use that same plot group name to create 2 or more plots. 
4. Save the configuration, go to the project page and select `Plots` from the left hand navigation menu
5. in the "jump to" selector, select a plot and observe the page jump to the location of that plot

### Checklist
